### ISSUE TITLE
YouTube video stuttering

### ISSUE THREAD

**Issue by grempe** [Ranchero-Software/NetNewsWire#3122]
Embedded YouTube videos are stuttering visually and in audio in NNW 6.0.2 on macOS. Tested for multiple videos. 

1gps Fios internet (symmetric)

See example:

https://share.icloud.com/photos/006l-oWcqkzM--y3fBTeXkI1Q

**Comment by brentsimmons** [comment]
We have no way to control video and audio — this is handled entirely by the system. Is there any chance something else was using a bunch of CPU on your machine?

**Comment by grempe** [comment]
Hi. No, cpu usage was low at the time. This was also why I compared in my video to Firefox playback which plays back cleanly. This was recorded from a Macbook Pro with 65gb RAM which has a wired ethernet connections and reports ~800mbps download speeds. 


I reproduced this with several feed items which had embedded YouTube videos. I have not experienced this outside of NNW. I had just updated NNW to latest version today.

**Comment by brentsimmons** [comment]
Unfortunately, we have no way to see in to what’s happening and we have no control over it. No way to affect it at all. This is handled entirely by Apple WebKit.

**Comment by grempe** [comment]
Follow up. I noticed this issue as well in the first-party Apple News app with the same kind of stuttering video. This eliminated NetNewsWire as the only culprity, and I assume Apple knows what it is doing with embeds. On a hunch I turned off my external audio device, forcing audio through my Macbook Pro speakers. The stuttering disappeared. When I changed the output back to my Motu M4 (which has its own driver as well) it returned.

Since this issues appears to be in the audio chain on my device I will close this issue. Thanks for the patience.
